src/topology/continuous_function/algebra.lean

```diff
@@ -113,6 +113,27 @@ instance {α : Type*} {β : Type*} [topological_space α] [topological_space β]
   ..continuous_map.semigroup,
   ..continuous_map.has_one }
 
+@[to_additive]
+instance {α : Type*} {β : Type*} [topological_space α]
+  [locally_compact_space α] [topological_space β]
+  [monoid β]  [has_continuous_mul β] : has_continuous_mul C(α, β) :=
+⟨begin
+  refine continuous_of_continuous_uncurry _ _,
+  change continuous ((λ (x : β × β), x.1 * x.2) ∘
+    (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1 x.2, x.1.2 x.2))),
+  have h1 : continuous (λ (x : (C(α, β) × C(α, β)) × α), (x.fst.fst) x.snd),
+  { change continuous ((λ (x : C(α, β) × α), x.1 x.2) ∘
+      (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1, x.2))),
+    refine continuous_ev.comp ((continuous_fst.comp continuous_fst).prod_mk continuous_snd), },
+  have h2 : continuous (λ (x : (C(α, β) × C(α, β)) × α), (x.fst.snd) x.snd),
+  { change continuous ((λ (x : C(α, β) × α), x.1 x.2) ∘ (λ (x : (C(α, β) × C(α, β)) × α),
+      (x.1.2, x.2))),
+    refine continuous_ev.comp ((continuous_snd.comp continuous_fst).prod_mk continuous_snd), },
+  refine (continuous_fst.comp (continuous.prod_mk _ _)).mul
+    (continuous_snd.comp (continuous.prod_mk _ _)),
+  all_goals { assumption, },
+end⟩
```

Comment:
Unless I'm missing something, there's already a proof for this with weaker assumptions further down in the file:
```suggestion
@[to_additive] instance {α : Type*} {β : Type*} [topological_space α]
  [topological_space β] [has_mul β] [has_continuous_mul β] : has_continuous_mul C(α, β) :=
{ continuous_mul := by
  { letI : uniform_space β := topological_group.to_uniform_space β,
    have : uniform_group β := topological_group_is_uniform,
    rw continuous_iff_continuous_at,
    rintros ⟨f, g⟩,
    rw [continuous_at, tendsto_iff_forall_compact_tendsto_uniformly_on, nhds_prod_eq],
    exactI λ K hK, ((tendsto_iff_forall_compact_tendsto_uniformly_on.mp filter.tendsto_id K hK).prod
      (tendsto_iff_forall_compact_tendsto_uniformly_on.mp filter.tendsto_id K hK)).comp'
      uniform_continuous_mul },
```
You can then remove the proof in the `topological_group` instance